src/Panel/panel.scss

```diff
@@ -92,6 +92,14 @@ $content-title-color: $brc-darkGrey !default;
     border-left: 1px solid $border-color;
   }
 
+  &.floating {
```

Comment:
This in particular shouldn't be a breaking change: new default style adds a border-bottom to the header, ending in a double border when collapsed. Old style didn't have border-bottom, so it should be a safe change. 

Not sure about the margin-left change few lines below (do we have some collapsable-left panel with the rotated title visible when collapsed)? 
Actually this change could be reverted and customized in the example